### PR TITLE
Failover addition

### PR DESCRIPTION
**Pull request by ConalCosgrove**
# Objectives: 
- ### Add config variables to allow bot to leverage new bdk failover functionality.
- ### Fix bug that caused UI to crash when `pageInstrumentBuilder` is undefined
- ### Fix bug that caused logs of services paged to not appear without a refresh for the user who pages out a team.

# Changes: 
- ## config.js
  - Added redis config variables 
  - removed perf config setting

- ## package.json
  - Update bdk and eslint plugins
- ## web/components/App.jsx
  - Added code to ensure that if `this.pageInstrumentBuilder` is undefined the UI won't crash
- ## web/index.js
  - Updated renderUI function to calculate incidents itself rather than being provided them. 
  - Removed repeated code that calculated incidents (which was moved inside renderUI)

### CODE REVIEW

**Comment by ConalCosgrove** [comment]
CI won't pass until new bdk patch is released [https://github.com/salesforce/refocus-bdk/pull/37](url)

**Comment by BrandonMurnane** [comment]
missing a readme update

**Comment by ConalCosgrove** [comment]
@BrandonMurnane classic, will add now

File: web/components/App.jsx [hunk 1]
```diff
@@ -58,8 +57,10 @@ class App extends React.Component{
       setTimeout(this.closeToast, TOAST_TIMEOUT);
     }
     if (prevProps.recommendations.length !== this.props.recommendations.length) {
+      const recommendationsToSet = this.props.recommendations.map(({ label }) => label);
+      // eslint-disable-next-line no-unused-expressions
       this.pageInstrumentBuilder
```
Comment (cassiodias) [inline web/components/App.jsx:62]: Why this guy sometimes is undefined?

Comment (ConalCosgrove) [reply web/components/App.jsx:62]: I believe it's a condition where it hasn't been initialised yet

**Pull request merged**